packages/serverless/settings/README.mdx

```diff
@@ -0,0 +1,10 @@
+---
+id: serverless/packages/settings
+slug: /serverless/packages/settings
+title: Serverless Advanced Settings
+description: A package of common settings for Serverless projects.
+tags: ['serverless', 'package']
+date: 2023-08-24
+---
+
+This package contains common settings for Serverless projects.
```

Comment:
nit: we should expand this with more detail.

Reply:
Sure, I added some details. Let me know if there are any other specific details that you think might be helpful here.